delta-api/src/main/java/io/cdap/delta/api/assessment/ColumnDetail.java

```diff
@@ -26,11 +26,15 @@
   private final String name;
   private final SQLType type;
   private final boolean nullable;
+  private final int length;
+  private final int scale;
 
-  public ColumnDetail(String name, SQLType type, boolean nullable) {
+  public ColumnDetail(String name, SQLType type, boolean nullable, int length, int scale) {
```

Comment:
are these things not present within the type?

This also won't be enough for other more complex types, such as geo types. It may be better to just have a Map<String, String> properties here, to let the TableRegistry set whatever it wants

Reply:
If you mean SQLType? Yes. these are not present with that, these information will only be bind with the metadata.

Adding a properties map is good call. It will be more flexible. Will do that.